modules/services/activate-system/default.nix

```diff
@@ -65,7 +65,7 @@ in
           ln -sfn /run/current-system /nix/var/nix/gcroots/current-system
         fi
 
-        ${config.system.activationScripts.etcChecks.text}
+        ${config.system.activationScripts.checks.text}
```

Comment:
The one thing I am concerned about here is if `checks` has stuff that needs to be run in a graphical context. Based on a quick search through our in-tree checks it seems like this change would cause errors for the activation daemon from `modules/users/default.nix` in some cases, it might cause errors for our other checks, and might for out-of-tree checks if people were assuming that `checks` would only run in a graphical session.

Reply:
The activation daemon only activates system configurations that were previously current. So in theory it should never be trying to delete users and so on. What other checks do you think might be problematic here?

Reply:
Ah, good point. I don't have any problematic checks I can think of, and they should be read-only anyway, so I'm good with this.